simpeg_drivers/constants.py

```diff
@@ -271,6 +278,19 @@
         "lineEdit": False,
         "enabled": True,
     },
+    "reference_model": {
+        "association": ["Cell", "Vertex"],
+        "main": True,
+        "dataType": "Float",
+        "group": "Mesh and models",
+        "isValue": True,
+        "optional": True,
+        "enabled": False,
+        "parent": "mesh",
+        "label": "Reference",
+        "property": None,
+        "value": 0.0,
+    },
```

Comment:
why the move?

Reply:
missed this change when sorting merge conflicts with 1774.  Moved it back on top of lower bound.